user_funcs.py

```diff
@@ -116,3 +117,246 @@ def login(conn) -> tuple[str, int]:
                 return results[0]
 
             print("Username or password incorrect!")
+#THis whole function does work correctly
+def create_collection(conn, user_ID) -> None:
+    """
+    creates a new collection for the user and puts the new collection
+    in the movie collection table
+
+    modifying the collection and listing the collection are not in this method
+
+    :param conn: The connection to the database to execute SQL statements
+    :param user_ID: The ID of the user for their movie collection
+    :return: Nothing
+    """
+    #A query that will insert the movie_ID the user selected into the incollection table with associated collection_ID
+    add_movie_collection_query = """
+    INSERT INTO incollection (movieid, collectionid)
+    VALUES(%s, %s)
+    """
+
+    #Query that gets the last collection ID in the moviecollection table
+    collection_ID_query = """
+    Select collectionid FROM moviecollection AS mc
+    ORDER BY mc.collectionid DESC LIMIT 1
+    """
+
+    collection_ID = 0
+    collection_Name = ""
+    loop_state = True
+    with conn.cursor() as curs:
+        user_ID = str(user_ID)
```

Comment:
NIT:
```suggestion
        user_id = str(user_id)
```

Reply:
I realized the above issue is in a few places. I can comment out each instance, but it's probably quicker for you to go through them.